app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -291,6 +291,7 @@ def parse_template(template)
         new_result[:container_template_parameters] = parse_template_parameters(template.parameters)
         new_result[:labels] = parse_labels(template)
         new_result[:objects] = template.objects.to_a.collect(&:to_h)
+        new_result[:type] = ManageIQ::Providers::Openshift::ContainerManager::ContainerTemplate.name
```

Comment:
Now I understand why this morning I saw refresh emit type = ManageIQ::Providers::ContainerManager::ContainerTemplate, because that's what this const lookup returned without the require_nested...  [details: https://github.com/ManageIQ/manageiq/pull/15523/files#r131175697]

Please use string 'ManageIQ::Providers::Openshift::ContainerManager::ContainerTemplate', less magic and for consistency with another place in this file.